src/app/components/header/header.component.html

```diff
@@ -0,0 +1,59 @@
+<header>
+    <div class="bg-light py-1">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6 col-12 text-center text-md-start"><span> Super Value Deals - Save more with
+                        coupons</span> </div>
+            </div>
+        </div>
+    </div>
+    <div class="navbar navbar-light py-lg-4 pt-3 px-0 pb-0">
+        <div class="container">
+            <div class="row w-100 align-items-center g-lg-2 g-0">
+                <div class="col-xxl-2 col-lg-3"> <a class="navbar-brand d-none d-lg-block" routerLink="/"> <img
+                            src="/assets/images/logo/freshcart-logo.svg" alt="eCommerce HTML Template"> </a>
+                    <div class="d-flex justify-content-between w-100 d-lg-none"> <a class="navbar-brand" routerLink="/">
+                            <img src="/assets/images/logo/freshcart-logo.svg" alt="eCommerce HTML Template"> </a>
+                        <div class="d-flex align-items-center lh-1">
+                            <button class="navbar-toggler collapsed pull-left" type="button" data-bs-toggle="collapse"
+                                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                                aria-expanded="false" aria-label="Toggle navigation"> <span
+                                    class="icon-bar top-bar mt-0"></span> <span class="icon-bar middle-bar"></span>
+                                <span class="icon-bar bottom-bar"></span>
+                            </button>
+                        </div>
+                    </div>
+                    <div class="navbar-collapse collapse" id="navbarSupportedContent">
+                        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                            <li *ngFor="let category of categories$ | async as categoryList" class="nav-item">
```

Comment:
Why you added "as categoryList" ?

Reply:
My mistake, I have refactored this code from different code block and I havent brushed it entirely.  Fixed it.